### DISCUSSION TITLE
Allow authors to request changes on their own pull request

### DISCUSSION THREAD

**Discussion by joel-coffman** [community/community#8918] [Pull Requests]
For personal (private) repositories, the author of pull requests cannot request changes ("Pull request authors cannot request changes on their own pull request"). This restriction makes sense insofar as the author can presumably make any requested changes directly, yet sometimes I find myself needing to indicate that a pull request should *not* be merged for some reason (e.g., an issue that requires significant refactoring). A comment to this effect provides a workaround, but a comment is easy to overlook, particularly when the conversation is long.

[Draft pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests) provide similar functionality (i.e., converting a pull request to draft until the issue has been resolved), but draft pull requests are not available for personal repositories.

**Comment by pandu-crunchbase** [comment]
I've definitely encountered this a number of times as well -- would love this to be allowed

**Comment by janbrasna** [comment]
That.

I often have to open PRs for others' work, as a way to actually start the review process… and it's frustrating I can't then request changes, all I can do is leave neutral comments then.

**Comment by bastiao** [comment]
It will be a nice feature to add, indeed!

**Comment by Anddymodera** [comment]
I've definitely encountered this a number of times as well -- would love this to be allowed
